src/Float/Extra.elm

```diff
@@ -0,0 +1,377 @@
+module Float.Extra exposing
+    ( aboutEqual
+    , toFixedDecimalPlaces, toFixedSignificantDigits, boundaryValuesAsUnicode
+    , range
+    )
+
+{-|
+
+
+# Equality
+
+@docs aboutEqual
+
+
+# Formatting Floats
+
+@docs toFixedDecimalPlaces, toFixedSignificantDigits, boundaryValuesAsUnicode
+
+
+# Ranges
+
+@docs range
+
+-}
+
+-- toFixedDecimalDigits implementation
+
+
+zeroes : Float -> String
+zeroes v =
+    String.repeat (floor v) "0"
+
+
+adjustDecimalPlace : Float -> Float -> Float
+adjustDecimalPlace x magnitude =
+    if magnitude < 0 then
+        x * 10 ^ -magnitude
+
+    else
+        x / 10 ^ magnitude
```

Comment:
I'm guessing there are precision issues with just using `x / 10 ^ magnitude` for both branches?

Reply:
I think so... Not really sure at this point 🤷